tools/generator/files/rules.cc

```diff
@@ -1,9 +1,13 @@
 // SPDX-License-Identifier: GPL-2.0-or-later
 #include "rules.hh"
+
+#include <memory>
+#include <utility>
+
 #include "actions.hh"
 
 Rules::Rules(const rules::Options opt)
-    :!!rules_cc_inheritance!! sandbox_(opt.time)
```

Comment:
Is that intended? I imagine this would break the template replacement.

Reply:
This was not intentional. Thanks.